### PR TITLE
feat: umi webpack should output string

### LINKED ISSUE #5900: umi webpack 输出字符串格式的配置
现在 umi webpack 输出的配置是 JS 对象，如果嵌套层级过多，就会变成 `[Object]`、`[Function]` 这样：
image
为了可读性，建议改成字符串格式。

### PR DESCRIPTION
**Pull request by yingzhiji**
##### Checklist

- [x] `npm test` passes
- [x] tests are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

- feat: umi webpack should output string
- close #5900

-----
[View rendered docs/docs/cli.md](https://github.com/yingzhiji/umi/blob/feat/improve-umi-webpack/docs/docs/cli.md)
[View rendered docs/docs/cli.zh-CN.md](https://github.com/yingzhiji/umi/blob/feat/improve-umi-webpack/docs/docs/cli.md)

### CODE REVIEW

File: packages/preset-built-in/package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "@umijs/server": "3.3.3",
     "@umijs/types": "3.3.3",
     "@umijs/utils": "3.3.3",
+    "cli-highlight": "^2.1.9",
```
Comment (sorrycc) [inline packages/preset-built-in/package.json:41]: 写死版本。

File: packages/preset-built-in/package.json [hunk 2]
```diff
@@ -55,6 +56,7 @@
     "regenerator-runtime": "0.13.5",
     "serialize-javascript": "5.0.1",
     "umi-webpack-bundle-analyzer": "3.6.0",
+    "webpack-chain": "^6.5.1",
```
Comment (sorrycc) [inline packages/preset-built-in/package.json:59]: 在 packages/bundler-webpack/src/index.ts 里导出 webpackChain，不要单独引。

**Comment by sorrycc** [comment]
给个改过之后的截图。

**Comment by sorrycc** [comment]
给个改过之后的截图。

File: yarn.lock [hunk 3]
```diff
@@ -5544,6 +5544,18 @@ cli-cursor@^3.1.0:
   dependencies:
     restore-cursor "^3.1.0"
 
+cli-highlight@^2.1.9:
+  version "2.1.9"
+  resolved "https://registry.npm.taobao.org/cli-highlight/download/cli-highlight-2.1.9.tgz?cache=0&sync_timestamp=1606814194837&other_urls=https%3A%2F%2Fregistry.npm.taobao.org%2Fcli-highlight%2Fdownload%2Fcli-highlight-2.1.9.tgz#4f4ecb05326d70d56d4b4249fabf9a70fb002497"
```
Comment (ycjcl868) [inline yarn.lock:5549]: 用 yarn，不要用 taobao 源，ci 机器上过不去

Comment (yingzhiji) [reply yarn.lock:5549]: yarn 官方的源？

**Pull request closed without merging**